src/data/faq.json

```diff
@@ -1542,6 +1542,38 @@
                             "On the contrary: the open-source community increases the level of security of the software, because the source code is open to all and anyone can download it and comb through it line by line.",
                             "We believe that security is not helped by a lack of transparency. That is why we are taking an open-source approach."
                         ]
+                    },
+                    {
+                        "title": "What is the 'In-App Reset' feature of the Corona-Warn-App for?",
+                        "anchor": "in_app_reset",
+                        "active": true,
+                        "textblock": [
+                            "If the Corona-Warn-App (CWA) does not work as expected, the In-App Reset function ('Reset Application') can be used to delete part of the personal data used by the CWA without compromising the functionalities of the CWA after resetting. It should be noted that during an in-app reset, the following data is not deleted:",
+                            "<ul><li>The time when onboarding was completed</li><li>Running error logs</li><li>For Apple iOS: the API token, so that the data donation or the function to participate in surveys from the RKI on certain events continues to work</li></ul>",
+                            "This data remains encrypted after an in-app reset on the smartphone.",
+                            "The Exposure Notification Framework (ENF) keys are managed by the operating system and can be deleted in the system settings of Android or iOS (See  <a href='#delete_all_data_cwa' target='_self' rel='noopener noreferrer'>See How can I delete all data stored in the CWA?</a>)."
+                        ]
+                    },
+                    {
+                        "title": "How can I delete all data stored in the Corona-Warn-App?",
+                        "anchor": "delete_all_data_cwa",
+                        "active": true,
+                        "textblock": [
+                            "By de-installing the Corona-Warn-App (CWA), all data of the CWA is deleted. However, a user should note that, for example, the data of the Exposure Notification Framework (ENF) is managed by the operating system of his smartphone. Therefore, this ENF data on Android is only deleted by a CWA deinstallation if no other application (app) has been installed on the user's smartphone that uses the ENF. On Apple iOS, the ENF data must be deleted manually because it will not be deleted by the CWA deinstallation.",

```

Comment:
"de-installing" => "uninstalling"
"a CWA deinstallation" => "uninstalling CWA"
"the CWA deinstallation" => "uninstalling CWA"

Android uses the term "uninstall"
https://dictionary.cambridge.org/dictionary/english/uninstall

Reply:
Fix will follow. Thanks.